src/components/Characters/Fox/Fox.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import FoxArtWork from './FoxArtWork';
```

Comment:
As per **readme**:
keep your imports order correct -- the higher a file is in directory structure, the higher on the list it should be, eg:
```
import React from 'react';
import PropTypes from 'prop-types';

import Headline from '../../Headline';
import { SugarCat } from './Characters';
```

I think it would be more pleasant to read if moods / genders could be close together.
For future reference maybe check if there is _linter rule_ to enforce it.

Reply:
Fixed in e3ef83fb68c81d0d81805008b3a523b0854a95e9
However, I would encourage to use *eslint* [sort-imports](https://eslint.org/docs/rules/sort-imports) as it provides more options for sorting which can be configured to match preferences. _Personally_, I use IDE support for auto-imports, so I don't pay much attention  to the order. I'd rather check for unused imports